### PR TITLE
ANDROID-4213 Vpn settings shortcut added + shortcuts refactor

### PR DESCRIPTION
**Pull request by stehlikr**

### CODE REVIEW

File: res/app/control-panes/dashboard/apps/apps-controller.js [hunk 1]
```diff
@@ -16,23 +16,13 @@ module.exports = function ShellCtrl($scope) {
   // TODO: Move this to server side
   // TODO: Android 2.x doesn't support openSetting(), account for that on the UI
 
-  function openSetting(activity) {
-    run('am start -a android.intent.action.MAIN -n com.android.settings/.Settings\\$' +
-    activity)
-  }
```
Comment (coufalja) [inline res/app/control-panes/dashboard/apps/apps-controller.js:22]: Why did you remove this?

**Review by coufalja — CHANGES_REQUESTED** [review]

File: res/app/control-panes/dashboard/apps/apps-controller.js (hunk 1 above)
Comment (stehlikr) [reply res/app/control-panes/dashboard/apps/apps-controller.js:22]: Ok I thought this was added by me/us. I will revert it then alongside with `openWiFiSettings` and `openManageApps` that are also part of original code.

**Review by coufalja — DISMISSED** [review]

**Review by coufalja — APPROVED** [review]
